cmd/main.go

```diff
@@ -9,7 +9,7 @@ import (
 	"os"
 
 	"github.com/btcsuite/btcd/btcec/v2"
-	sphinx "github.com/lightningnetwork/lightning-onion"
+	sphinx "github.com/gijswijs/lightning-onion"
```

Comment:
why can't we use a proper commit has of main instead of replacing the whole lib ? you should rather do a relativ replace in go.mod and then that's it.

Reply:
This was only a temp commit. After rebase it was dropped.